packages/pages/src/vite-plugin/build/closeBundle/manifest.ts

```diff
@@ -1,11 +1,11 @@
 import fs from "fs-extra";
 import path from "path";
 import { ProjectStructure } from "../../../common/src/project/structure.js";
-import { TemplateModuleCollection } from "../../../common/src/template/internal/loader.js";
 import { convertToPosixPath } from "../../../common/src/template/paths.js";
 import { Manifest } from "../../../common/src/template/types.js";
 import { glob } from "glob";
 import { Path } from "../../../common/src/project/path.js";
+import { TemplateModuleCollection } from "../../../common/src/template/loader/loader.js";
```

Comment:
Why all these import shuffling changes?

Reply:
Not sure. Probably just me removing and adding things back. There's no format for actual import ordering.